src/auth/components/SignUpForm.tsx

```diff
@@ -0,0 +1,213 @@
+              Address
+            </Label>
+            <Input
+              placeholder="Enter your Address"
+              id="address"
+              className="h-10"
+            />
+          </div>
+        </div>
+
+        {/* Age & Gender */}
+        <div className="flex gap-3">
+          <div className="space-y-3 w-full">
+            <Label htmlFor="age" className="text-gray-700">
+              Age
+            </Label>
+            <Input
+              placeholder="Enter your Age"
+              id="age"
+              className="h-10"
+              type="number"
+            />
+          </div>
+          <div className="space-y-3 w-full">
+            <Label htmlFor="gender" className="text-gray-700">
+              Gender
+            </Label>
+            <Select>
+              <SelectTrigger id="gender" className="w-full py-[18.5px]">
+                <SelectValue placeholder="Gender" />
+              </SelectTrigger>
+              <SelectContent>
+                <SelectItem value="male">Male</SelectItem>
+                <SelectItem value="female">Female</SelectItem>
+                <SelectItem value="other">Other</SelectItem>
+              </SelectContent>
+            </Select>
+          </div>
+        </div>
+
+        {/* Contact Number & Profile Image */}
+        <div className="flex gap-3">
+          <div className="space-y-3 w-full">
+            <Label htmlFor="contact-number" className="text-gray-700">
+              Contact Number
+            </Label>
+            <Input
+              placeholder="Enter your Contact Number"
+              id="contact-number"
+              className="h-10"
+              type="number"
+            />
+          </div>
+          <div className="space-y-3 w-full">
+            <Label htmlFor="picture" className="text-gray-700">
+              Profile Picture
+            </Label>
+            <Input
+              id="picture"
+              type="file"
+              className="h-10 text-gray-700 cursor-pointer"
+            />
+          </div>
+        </div>
+
+        {/* BirthDate & Password */}
+        <div className="flex gap-3">
+          <div className="space-y-3 w-full">
+            <Label htmlFor="birth-date" className="text-gray-700">
+              Birth Date
+            </Label>
+            <Popover open={open} onOpenChange={setOpen}>
+              <PopoverTrigger asChild>
+                <Button
+                  type="button"
+                  id="birth-date"
+                  variant="outline"
+                  className="w-full justify-start font-normal cursor-pointer h-10"
+                >
+                  <CalendarIcon className="mr-2 h-4 w-4" />
+                  Select Your Birth Date
+                </Button>
+              </PopoverTrigger>
+              <PopoverContent className="w-auto p-0" align="start">
+                <Calendar mode="single" initialFocus fromDate={new Date()} />
```

Comment:
By mistake you added `formDate`. 😅

```suggestion
                <Calendar mode="single" initialFocus />
```

Reply:
I have removed `formDate` (https://github.com/krushna-simform/05-e-commerce/pull/2/commits/313e01a4742d9f1972898c5596d776be659aea7e)